packages/expo/bin/cli.ts

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+import arg from 'arg';
+import chalk from 'chalk';
+
+const defaultCmd = 'config';
+
+export type Command = (argv?: string[]) => void;
+
+const commands: { [command: string]: () => Promise<Command> } = {
+  // Add a new command here
+  config: () => import('../cli/config').then((i) => i.expoConfig),
+};
+
+const args = arg(
+  {
+    // Types
+    '--version': Boolean,
+    '--help': Boolean,
+
+    // Aliases
+    '-v': '--version',
+    '-h': '--help',
+  },
+  {
+    permissive: true,
+  }
+);
+
+if (args['--version']) {
+  // Version is inlined into the file using taskr build pipeline
+  console.log(`expo v${process.env.__EXPO_VERSION}`);
```

Comment:
This might be breaking for just a few Expo users that rely on `expo --version` returning only the semver numbers.

Reply:
This will be `npx expo --version` but we can change it so it shouldn't be breaking, but we can change it to be more normalized.